README.md

```diff
@@ -5,38 +5,42 @@
   <a href="https://circleci.com/gh/fhir-sci/hl7-fhir-ig-publisher"><img src="https://img.shields.io/circleci/project/github/fhir-sci/hl7-fhir-ig-publisher.svg?style=for-the-badge"></a>
 </p>
 
-This repository is a fork of https://github.com/FHIR/latest-ig-publisher.
+The purpose of this repo to Dockerize the HL7 IG Publisher jar so that you can avoid installing dependencies and run the publisher in a Docker container.
```

Comment:
```suggestion
The purpose of this repo is to Dockerize the HL7 IG Publisher jar so that you can avoid installing dependencies and run the publisher in a Docker container.
```